conf/distro/nilrt.conf

```diff
@@ -25,3 +25,23 @@ KERNEL_IMAGEDEST = "boot/runmode"
 ROOT_HOME = "/home/admin"
 
 IMAGE_LINGUAS ?= "en-us en-us.iso-8859-1"
+
+BUILD_IMAGES_FROM_FEEDS = "1"
+
+# Get NIOE packages from local deploy feed and NI packages from web feed
+IPK_NIOE_SUBFEED_URI ?= "file://${DEPLOY_DIR_IPK}"
+
+FEED_SRC ?= "download.ni.com/ni-linux-rt/feeds"
+NI_MAIN_FEED_VERSION ?= "2020.5"
+NI_SALT_FEED_VERSION ?= "20.0"
+IPK_NI_MAIN_SUBFEED_URI ?= "http://${FEED_SRC}/${NI_MAIN_FEED_VERSION}/ni-main/"
```

Comment:
Is it true that the image which you build from this patchset won't have an opkg config line for the `ni-main` feed source? I think it won't because you haven't asked the package_manager bbclass to write one in the rootfs with either the `PACKAGE_FEEDS*` variables (upstream-standard) or the `PACKAGE_FEED_URIS_ADDITIONAL` (nilrt-specific).

If that's the case, you probably want to [add a line](https://github.com/ni/meta-nilrt/blob/nilrt/master/dunfell/conf/distro/nilrt.inc#L130) to `nilrt.inc` like
```
PACKAGE_FEED_URIS_ADDITIONAL[ni-main] = "${NILRT_FEEDS_URI_RELEASE}/ni-main"
```

Reply:
Yes, the image does not have that config line. This is currently added by the conf [files](https://dev.azure.com/ni/DevCentral/_git/ni-central?version=GCea6985a18c9614ca87ba416cb4a71da550f2014e&path=%2Fsrc%2Frtos%2FSystemImaged%2Fnilrt_onert_system_image%2Fsource%2Frss%2Fcdf%2Fni-software-cg.conf) in SystemImage. Do you want to add it to the base image instead?

Reply:
Yeah; any feed from which we install IPKs during image creation needs to be represented in the opkg `.conf` files.